<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gui::UnitInfoWindow</class>
 <widget class="QMainWindow" name="gui::UnitInfoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>307</width>
    <height>497</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>230</width>
    <height>100</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#wdg_scroll_area_content {
	background-color:  #F5F5F5;
}
</string>
  </property>
  <widget class="QWidget" name="wdg_main_widget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="wdg_scroll_area_content">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>307</width>
         <height>497</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="wdg_scroll_area_content_layout" stretch="0,0,0,1,0">
        <property name="margin">
         <number>6</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>6</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
            <item>
             <layout class="QGridLayout" name="gridLayout_3">
              <property name="horizontalSpacing">
               <number>15</number>
              </property>
              <property name="verticalSpacing">
               <number>6</number>
              </property>
              <item row="2" column="2">
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="lbl_weight">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: #4B4B4B;</string>
                </property>
                <property name="text">
                 <string>Weight</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="lbl_unit_value">
                <property name="text">
                 <string>(16, 22)</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLabel" name="lbl_weight_value">
                <property name="text">
                 <string>23.12423</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0" colspan="2">
               <widget class="QGroupBox" name="grp_outputs">
                <property name="styleSheet">
                 <string notr="true">QGroupBox::tab {
	color: #4B4B4B;
}</string>
                </property>
                <property name="title">
                 <string>Outputs</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_2">
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>9</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>3</number>
                 </property>
                 <property name="horizontalSpacing">
                  <number>10</number>
                 </property>
                 <item row="0" column="0">
                  <widget class="QLabel" name="lbl_output_pos">
                   <property name="text">
                    <string>Positive</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="lbl_output_neg">
                   <property name="text">
                    <string>Negative</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLabel" name="lbl_output_pos_value">
                   <property name="text">
                    <string>123 (90%)</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QLabel" name="lbl_output_neg_value">
                   <property name="text">
                    <string>12 (10%)</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="lbl_unit">
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: #4B4B4B;</string>
                </property>
                <property name="text">
                 <string>Unit</string>
                </property>
               </widget>
              </item>
              <item row="3" column="2">
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="horizontalLayout_2" stretch="0,1">
              <item>
               <widget class="QFrame" name="frm_unit_picture">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>80</width>
                  <height>80</height>
                 </size>
                </property>
                <property name="frameShape">
                 <enum>QFrame::Box</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Plain</enum>
                </property>
                <layout class="QVBoxLayout" name="frm_cell_paint_layout">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="margin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="lbl_unit_pixmap">
                   <property name="styleSheet">
                    <string notr="true">background-color: #FFFFFF;</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                   <property name="indent">
                    <number>0</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Minimum</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>10</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="lbl_atom_symbol">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #4B4B4B;</string>
          </property>
          <property name="text">
           <string>Atom Symbol</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tbl_atom_symbol">
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="showGrid">
           <bool>true</bool>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="rowCount">
           <number>2</number>
          </property>
          <property name="columnCount">
           <number>2</number>
          </property>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <row/>
          <row/>
          <column/>
          <column/>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="wdg_other_labels" native="true">
          <property name="styleSheet">
           <string notr="true">QToolBox::tab {
	background: qlineargradient(
			x1: 0, y1: 0, x2: 0, y2: 1, 
			stop: 0 #E1E1E1, stop: 0.4 #DDDDDD, 
			stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
	border-radius: 3px;
	color: #4B4B4B;
 }

QToolBox::tab:pressed {
	background: darkgray;
 }

</string>
          </property>
          <layout class="QVBoxLayout" name="wdg_other_labels_layout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QToolBox" name="tbx_other_labels">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="wdg_label_1_page">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>295</width>
                <height>79</height>
               </rect>
              </property>
              <attribute name="label">
               <string>Page 1</string>
              </attribute>
              <layout class="QVBoxLayout" name="wdg_label_1_page_layout">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="margin">
                <number>0</number>
               </property>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btn_close">
            <property name="text">
             <string>Close</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_close</sender>
   <signal>clicked()</signal>
   <receiver>gui::UnitInfoWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>539</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
